server/komodo-rest/src/main/java/org/komodo/rest/relational/response/RestVdb.java

```diff
@@ -195,7 +205,7 @@ public void setConnectionType(String connectionType) {
      */
     public int getVersion() {
         Object version = tuples.get(VERSION_LABEL);
-        return version != null ? Integer.parseInt(version.toString()) : null;
+        return version != null ? Integer.parseInt(version.toString()) : VERSION_DEFAULT;
     }
```

Comment:
Same as original path for these values. If the item is null then its not included in the json content so the network transfer is smaller.